README.md

```diff
@@ -26,7 +168,7 @@ Information on how to install of the VOLTTRON platform can be found
 Install and start the volttron-platform-driver.
 
 ```shell
-vctl install volttron-platform-driver --vip-identity platform.driver --start
+vctl install volttron-platform-driver --vip-identity platform.driver
```

Comment:
--start?

Reply:
If you start it first, and then configure, your configs won't take until you restart. Even once that is corrected, you would be rescheduling every time you add add a device to the configs. This should not have --start.